packages/calcite-components/src/components/panel/panel.tsx

```diff
@@ -525,27 +511,16 @@ export class Panel
   };
 
   renderContent(): VNode {
-    const { hasFab } = this;
-
-    const defaultSlotNode: VNode = <slot key="default-slot" />;
-    const containerNode = hasFab ? (
-      <section class={CSS.contentContainer}>{defaultSlotNode}</section>
-    ) : (
-      defaultSlotNode
-    );
-
     return (
       <div
-        class={{
-          [CSS.contentWrapper]: true,
-          [CSS.contentContainer]: !hasFab,
-          [CSS.contentHeight]: hasFab,
-        }}
+        class={CSS.contentWrapper}
         onScroll={this.panelScrollHandler}
         // eslint-disable-next-line react/jsx-sort-props
         ref={this.setPanelScrollEl}
       >
-        {containerNode}
+        <section class={CSS.contentContainer}>
```

Comment:
@asangma do you remember why this `section` was necessary for the FAB? Is there any harm in always having it there for consistency and so the default slot doesn't get rearranged?

Reply:
I think the section was there to keep the FAB sticky. But if it can be sticky without the section, that's cool too.

Reply:
Ok. i have it so the section is always there regardless of the FAB or not. Ill move forward with that since its safer.